src/routes/trip-members.js

```diff
@@ -32,8 +33,20 @@ export function sendToWaitlist (req, res) {
   const owner = req.db.get('SELECT owner FROM trips WHERE id = ?', tripId).owner
   if (userId === owner) return res.sendStatus(400)
 
-  req.db.run('UPDATE trip_members SET pending = 1 WHERE trip = ? and user = ?', tripId, userId)
+  /**
```

Comment:
Total style nitpick, but I think you're over-explaining yourself here. Consider:
```js
  // Remove then re-add member so that they're at the back of the auto-approval queue
  req.db.run('DELETE FROM trip_members WHERE trip = ? and user = ?', tripId, userId)
  req.db.run('INSERT INTO trip_members (trip, user, leader, pending) VALUES (?, ?, ?, ?)', tripId, userId, 0, 1)

  autoApprove(tripId)(req.db)
```

Reply:
Now that I think about it, the UX is going to be pretty bad here if you're sending people to the waitlist but the auto-approval is on and it adds them right back. What do you think is the right solution to that? I think maybe we should disable "send to waitlist" entirely if the auto-approval is on.

Reply:
I have thought about this more and let's do it this way: can't send anyone to the waitlist if you're auto-approving. Makes everything cleaner, less likely to have unexpected behavior.